docs/src/add-services/mysql/_index.md

```diff
@@ -42,26 +42,32 @@ Dedicated environments only support the InnoDB storage engine.
 Tables created on Dedicated environments using the MyISAM storage engine don't replicate between all hosts in the cluster.
 See how to [convert tables to the InnoDB engine](#storage-engine).
 
+{{% deprecated-versions %}}
+
+| **`mariadb`** | **`mysql`** | **`oracle-mysql`** |
+|----------------------------------|---------------|-------------------------|
+|  {{< image-versions image="mariadb" status="deprecated" >}} | {{< image-versions image="mariadb" status="deprecated" >}} | {{< image-versions image="oracle-mysql" status="deprecated" >}} |
+
 ### Switching type and version
 
 If you change the service type, your data is removed.
```

Comment:
:hammer_and_wrench:

See additional notes for more context.
```suggestion
If you change the service type or its name, your data is removed.
```

Reply:
I'd wait for #2331 to deal with that idea.